src/odemis/gui/comp/overlay/base.py

```diff
@@ -49,14 +49,15 @@
 class Label(object):
     """ Small helper class that stores label properties """
 
-    def __init__(self, text, pos, font_size, flip, align, colour, opacity, deg):
+    def __init__(self, text, pos, font_size, flip, align, colour, opacity, background, deg):
```

Comment:
Please add a docstring to indicate the type of background (None or some colour??).
I'd also recommend putting it last, with a default value `=None` so that you don't need to update every code using the Label.